src/odemis/acq/stream/_base.py

```diff
@@ -348,11 +348,11 @@ def _va_sync_setter(self, origva, v):
         return: the real new value (as accepted by the original VA)
         """
         if self.is_active.value:  # only synchronised when the stream is active
-            logging.debug("updating VA (%s) to %s", origva, v)
+            logging.debug(u"updating VA (%s) to %s", origva, v)
```

Comment:
might be nice to have all logging debugs consistently starting with a capital letter.